pkg/dbutil/common.go

```diff
@@ -169,48 +169,55 @@ func GetRowCount(ctx context.Context, db *sql.DB, schemaName string, tableName s
 	return cnt.Int64, nil
 }
 
-// GetRandomValues returns some random value of a column.
-func GetRandomValues(ctx context.Context, db *sql.DB, schemaName, table, column string, num int64, min, max interface{}, limitRange string, collation string) ([]interface{}, error) {
+// GetRandomValues returns some random value and these value's count of a column, just like sampling. Tips: args is the value in limitRange.
+func GetRandomValues(ctx context.Context, db *sql.DB, schemaName, table, column string, num int, limitRange string, collation string, args []interface{}) ([]string, []int, error) {
```

Comment:
maybe rename `args` to `limitArgs` is clearer? and maybe we can put it near `limitRange`?

Reply:
done